include/dlaf/multiplication/general/impl.h

```diff
@@ -19,13 +19,52 @@
 #include <dlaf/communication/communicator_grid.h>
 #include <dlaf/matrix/distribution.h>
 #include <dlaf/matrix/index.h>
+#include <dlaf/matrix/matrix.h>
+#include <dlaf/matrix/matrix_ref.h>
 #include <dlaf/matrix/panel.h>
 #include <dlaf/multiplication/general/api.h>
 #include <dlaf/sender/when_all_lift.h>
+#include <dlaf/util_matrix.h>
 
 namespace dlaf::multiplication {
 namespace internal {
 
+template <Backend B, Device D, class T>
+void General<B, D, T>::callNN(const blas::Op opA, const blas::Op opB, const T alpha,
```

Comment:
We might also go for not exposing `blas::Op` at all for the moment, since we don't provide the implementation for anything other than NoTrans/NoTrans.